### PR TITLE
New Footer page

### PR DESCRIPTION
**Pull request by coreyvickery**
New Footer page being added to ux.redhat.com

### CODE REVIEW

File: docs/components/footer.njk [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+
+Temporary Warning alert until web component release
+<p style="background-color: #fdf7e7; border-top: 2px solid #f0ab00; padding: 16px 16px 16px 24px; font-size: 14px; letter-spacing: 0.2px;"><span style="color:#795600"><strong>Warning</strong></span><br />Text here</p>
+
+-->
+
+{% import 'component/components.njk' as components %}
+
+{% call components.section("Overview") -%}
+  <p>The footer is a container that displays links and content to visitors who reach the bottom of a page, it also shows them they are using an official Red Hat website.</p>
```
Comment (marionnegp) [inline docs/components/footer.njk:24]: I'd suggest splitting this into two sentences instead of using a comma.

File: docs/components/footer.njk [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+
+  <div class="margin-bottom--7">
+    <img src="{{ '/assets/footer/footer-style-layout.png' | url }}" alt="Footer component styling - layout" style="--inline-img-max-width: 992px;">
+  </div>
+
+{%- endcall %}
+
+{% call components.section("Usage") -%}
+
+  <p>The footer should be used to provide a visitor with additional links and content if they did not find what they were looking for, it can also include copyright and legal information.</p>
```
Comment (marionnegp) [inline docs/components/footer.njk:118]: I'd suggest splitting this into two sentences instead of using a comma.

File: docs/components/footer.njk [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+
+  <p>Do not stack the Global footer on top of the Modular footer.</p>
+
+  <div class="margin-bottom--7">
+    <img src="{{ '/assets/footer/footer-best-practice-1.png' | url }}" alt="Footer component best practice - changing the stacking order" style="--inline-img-max-width: 1000px;">
+  </div>
+
+  <h4>Using an accordion</h4>
+
+  <p>Do not use an accordion inside of a large Modular footer, only Modular footers for small screen sizes include an accordion.</p>
```
Comment (marionnegp) [inline docs/components/footer.njk:357]: I'd suggest splitting this into two sentences instead of using a comma.

**Review by marionnegp — COMMENTED** [review]
I found some potential punctuation changes, but everything else looks good! Images match headings, and links went to the right places.

**Review by ronniemcbride — APPROVED** [review]
Usually, your first instance of a word that will be referenced in abbreviation should be written out and then followed by the abbreviation in parentheses. than Anywhere else, after that, you just can use the abbreviation.

_ie:_ 

**Modular navigation**
A collection of links unique to the website information architecture (AI).

after that, you can just say **AI** throughout the rest of the page.

Other than that everything else looks good to me.

**Comment by bennypowers** [comment]
Does this supercede #327 ?

**Comment by coreyvickery** [comment]
@bennypowers Sorry, I forgot that you were working on [327](https://github.com/RedHat-UX/red-hat-design-system/pull/327). Let's move forward with this PR, but this won't happen again.

**Pull request merged**